commands/doit.go

```diff
@@ -125,35 +106,24 @@ func initConfig() {
 	viper.SetDefault("context", "default")
 }
 
-func findConfig() (string, error) {
-	if cfgFile != "" {
-		return cfgFile, nil
-	}
-
-	if homeDir() != "" {
-		legacyConfigPath := filepath.Join(homeDir(), ".doctlcfg")
-		if _, err := os.Stat(legacyConfigPath); err == nil {
-			msg := fmt.Sprintf("Configuration detected at %q. Please move .doctlcfg to %s",
-				legacyConfigPath, configPath())
-			warn(msg)
-		}
+func currentUser() *user.User {
+	var err error
+	if me != nil {
```

Comment:
The go authors thought of everything and saved you from having to memoize this yourself:

https://github.com/golang/go/blob/dfbc9c83a910c79cb3cc34dbfaed3c436e1b6ecb/src/os/user/lookup.go#L15

Reply:
nice to know, although now irrelevant :)